api/v1alpha1/limitador_types.go

```diff
@@ -33,12 +33,17 @@ type LimitadorSpec struct {
 
 	// +optional
 	Version *string `json:"version,omitempty"`
+
+	// +optional
+	Listener Listener `json:"listener,omitempty"`
 }
 
 // LimitadorStatus defines the observed state of Limitador
 type LimitadorStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
+
+	ServiceURL string `json:"service-url,omitempty"`
```

Comment:
If the case is to have gRPC and HTTP URL, we could have it like:

```
type Service struct {
  GrpcURL string `json:"grpc-url,omitempty"`
  HttpURL string `json:"http-url,omitempty"`
}

Reply:
Or, maybe provide the Host and Protocols/Ports separated? I fear that won't have much value other than checking the Spec

Reply:
Maybe to do this is separated PR's

What would I expect in the CR ''.status''

```go
// LimitadorStatus defines the observed state of Limitador      
type LimitadorStatus struct {
   // for exmple: myservice                                   
   ServiceName *string

  // for example "http://myservice.mynamespace.svc.cluster.local:8081"
   AdminEndpointURL *string

   // for example: 8080
   GRPCPort *int
   
  // Represents the observations of a foo's current state.                                                                                                    
  // Known .status.conditions.type are: "Available"                                                                                                           
  // +patchMergeKey=type                                                                                                                                      
  // +patchStrategy=merge                                                                                                                                     
  // +listType=map                                                                                                                                            
  // +listMapKey=type                                                                                                                                         
  Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,1,rep,name=conditions"`
}
```